care/emr/resources/account/spec.py

```
@@ -79,8 +79,14 @@ def perform_extra_serialization(cls, mapping, obj):
         mapping["patient"] = PatientListSpec.serialize(obj.patient)
 
 
-class AccountRetrieveSpec(AccountReadSpec):
+class AccountRetrieveSpec(AccountMinimalReadSpec):
     """Account retrieve specification"""
 
+    patient: dict
     cached_items: list = []
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)